### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by poperigby** [wezterm/wezterm#1142] [General]
I was thinking that the logo is a bit too cluttered with the menu buttons on-top. I also think it would be cool if it had simple terminal background like Alacritty's logo.
![image](https://user-images.githubusercontent.com/20866468/133163498-c82bb7e8-7fda-4203-994b-71e4222d9f02.png)

**Comment by wez** [comment]
I'm open to seeing alternative designs for the logo if someone has the right confluence of design skills and available time to contribute.

I don't want to eg: just take the alacritty logo and put a W in there because it minimizes the effort of the person(s) that designed the alacritty logo, and would be pretty obvious and unoriginal.

**Reply by poperigby** [reply]
Fair enough. Just wanted to get a discussion going. I have no design skills myself, but I still thought I could weigh in.

**Comment by valpackett** [comment]
Fun fact, for a little experiment I did with putting alacritty into a GTK shell a while ago, I made a [little variation on the gnome-terminal icon](https://github.com/unrelentingtech/galacritty/blob/83ec1ed87281b4adf040b8f0a6f712c2db854cb6/res/icons/hicolor/scalable/apps/technology.unrelenting.galacritty.svg) and then [the designer of the new gnome-terminal icon has contributed a redesign of my thing](https://github.com/unrelentingtech/galacritty/blob/4cdf382e2444c9d51b2d46fb34d780bcbd9cd210/res/icons/hicolor/scalable/apps/technology.unrelenting.galacritty.svg) due to [a GNOME initiative](https://gitlab.gnome.org/GNOME/Initiatives/-/issues/2). I kinda have a little silly similar idea now…

**Reply by poperigby** [reply]
I'm interested to see where that goes!

**Reply by valpackett** [reply]
![wezterm-1](https://user-images.githubusercontent.com/208340/134072191-a324b5c4-728b-459c-a03d-ad94dd7ee57f.png)

**Reply by poperigby** [reply]
That looks great! Maybe put a dollar sign before the W, and maybe try a different color for the W.

**Reply by valpackett** [reply]
![wezterm-2](https://user-images.githubusercontent.com/208340/134073440-008476dc-dcf2-4734-b4c7-0eabe622dba3.png)

Honestly I like the first one better, the dollar makes it look overloaded and kinda like a weird ascii smiley face type thing

**Reply by poperigby** [reply]
I like that color. Maybe you could try rotating the W, to see how that looks. @wez do you maybe want to weigh in?

**Reply by valpackett** [reply]
The whole concept was that it's a rotated "W" that's actually just made out of the two `>` prompt signs of the original gnome-terminal icon :)

**Reply by poperigby** [reply]
> Honestly I like the first one better, the dollar makes it look overloaded and kinda like a weird ascii smiley face type thing

Yeah, you're right. But keep the color I think.

**Reply by poperigby** [reply]
> The whole concept was that it's a rotated "W" that's actually just made out of the two > prompt signs of the original gnome-terminal icon :)

Ohhh, clever. Yeah, I think it's just the dollar sign that makes it look weird. I like the rotated W.

**Reply by valpackett** [reply]
Agree.

![wezterm-3](https://user-images.githubusercontent.com/208340/134074525-a72cfaa2-ec90-47b3-a8e5-75afd64ca502.png)

Bonus (somewhat poorly done) fun version:

![wezterm-4](https://user-images.githubusercontent.com/208340/134075937-ac1d11cb-b548-42c0-af76-e435755f3715.png)

**Reply by poperigby** [reply]
That is very fun. Do you think the non-fun (:cry: ) one is missing something? It looks kind of lonely.

**Reply by valpackett** [reply]
idk, I like the minimalism. I like both of the last two :D

**Reply by poperigby** [reply]
Same. You could make the W bigger maybe?

**Reply by valpackett** [reply]
The classic "make the logo bigger"? hehe. It's already quite tall, vertical space would feel cramped if it's enlarged.

**Reply by poperigby** [reply]
Even just making the W a bit taller and wider?

**Reply by follower** [reply]
Ooo, a bike shed! :D

I, too, initially missed the intention of the "two >" concept but do really like it as a concept.

How about leaning into the concept and preserve the space/gap between the `>` characters rather than joining them?

Hopefully it won't result in confusion with the terminal versions of `vvvv` or `vvvvv`. :D

**Comment by ghifarit53** [comment]
Hi, actually I was thinking about opening a new discussion a few days ago for an icon idea, but just now I found this discussion so I think I don't have to. So I use this terminal on macOS 12 and there's the new icon design that has rounded rectangle shape and somewhat has depth on it (since macOS 11), but I'm a bit bothered by the default WezTerm icon not following that design 😅 and I couldn't find one online whether someone has created icon that follows the new design, so I thought about creating the icons myself (based on this [Figma template](https://www.figma.com/community/file/857303226040719059), licensed under CC BY 4.0). I have been using it for like two weeks now and am happy so far 😄 So here's the results

<img width="119" alt="wezterm-icon-21" src="https://user-images.githubusercontent.com/59758342/147101408-1cb93c40-bb3e-4eb8-a7a9-74d233244679.png"> <img width="119" alt="wezterm-icon-2" src="https://user-images.githubusercontent.com/59758342/147101419-ca75165c-f8e4-4c24-a3c5-79dbbafc4b8b.png"> <img width="119" alt="wezterm-icon-3" src="https://user-images.githubusercontent.com/59758342/147101431-6fdd20b3-1f97-4f13-a52d-ad387e025c49.png">

I made 3 icons variation. The first one is resembling the original icon, then I make it more minimal on the second (and currently my favorite) by changing the title bar color to the same color as the terminal content, then the last one is only the `$W` symbol left. Added some depths and gradients here and there. Also, I don't really have the rights to use Operator Mono (the same font on the original icon and my icon) for the `$W` symbol and might change it to other font, but it's there just to give idea. Any feedback or suggestion are muchly appreciated, and finally thanks so much for this awesome terminal

**Reply by wez** [reply]
I think I also like the second one the most out of those options. I'd happily accept a PR that adds all three of those as svg files to this location in the repo: https://github.com/wez/wezterm/tree/main/assets/icon

**Reply by poperigby** [reply]
These are beautiful! I prefer the third one, but the second one is also nice. Could these replace the default icon?

**Reply by ghifarit53** [reply]
Also do you want to have some change on the font for the `$W` symbol? Because as I have said I don't really have the rights to use Operator Mono font, and I'm a little worried that it will bring any issue

**Reply by wez** [reply]
Using those existing exported outlines from Operator Mono in a logo as they appear in the current SVG shouldn't require any additional license overhead per https://www.typography.com/faq/100. You only need the license to use the font file itself, so if you're really just moving resizing my exported outlines, you should also be fine.

I'd like to keep using Operator Mono for the `$W`.

**Reply by ghifarit53** [reply]
That's good to know! gonna continue with my PR then

EDIT: Done!

**Reply by ghifarit53** [reply]
Thanks for the merge!

@lumbo7332 The icon is already replaced 😄

**Reply by wez** [reply]
Thanks; the new logo is available in the latest nightly builds!

**Reply by slmjkdbtl** [reply]
Sorry for reviving this thread but just my 2 cents: I think these icons look really nice and really appreciate the work, but personally prefer the original icon and think it has more personality. Same goes with the new fancy tab bar, it's nice but loses some personality comparing to the old purple one (love the arbitrary color choice and minimal style). These quirky defaults is what attracted me in the first place, it feels like a personal tool that embodies a person's will and taste, not a generic one. Not requesting any change since I can change them for myself on my device, but just throwing some thoughts out there :D

**Reply by valpackett** [reply]
@slmjkdbtl you might enjoy the [fun version](https://github.com/wez/wezterm/discussions/1142#discussioncomment-1359454) from above (btw, here's the [SVG](https://dl.unrelenting.technology/org.wezfurlong.wezterm.svg)) :)

**Comment by boonyasukd** [comment]
In case this discussion is still ongoing, I would like to add one suggestion to those who might be looking into redesigning the logo.

The current logo doesn't fare so well in light OS theme, as the dark blue characters ('$W') have an extremely low contrast with the black background. It's quite important to note that good logo design uses color contrast to its advantage (readability & standout).

When situated next to other icons that utilize color contrast well (like Windows Terminal), the Wezterm terminal logo is dim to the point where it looks almost like a black square. Changing the color of the text, or adding a bright character outline, should improve this.

![image](https://github.com/wez/wezterm/assets/8109816/addddbce-b498-4cf2-bff7-5bea41bce544)

**Reply by ty-davis** [reply]
I felt the same way so I made two new versions of the logo. I made a dark version where the "$W" is lightened up a little bit and a light version with a white background. The current wezterm logo is on the left and the new ones are in the middle and on the right.
![image](https://github.com/wez/wezterm/assets/118846857/9215739d-3c09-48e8-84c3-88e7c41b35ca)

I made these super quick so I might be up to tweaking it a little bit more later, but for now I don't mind the look of it. I like the light one but I don't think it really captures the spirit of a terminal application. ¯\_(ツ)_/¯
